tests/UI/campaigns/productV2/functional/02_CRUSStandardProduct.ts

```diff
@@ -0,0 +1,251 @@
+// Import utils
+import helper from '@utils/helpers';
+import type {BrowserContext, Page} from 'playwright';
+import {expect} from 'chai';
+
+// Import test context
+import testContext from '@utils/testContext';
+
+// Import pages
+import dashboardPage from '@pages/BO/dashboard';
+import createProductsPage from '@pages/BO/catalog/productsV2/add';
+import productsPage from '@pages/BO/catalog/productsV2';
+import foProductPage from '@pages/FO/product';
+import basicHelper from '@utils/basicHelper';
+
+// Import common tests
+import loginCommon from '@commonTests/BO/loginBO';
+import {enableNewProductPageTest, disableNewProductPageTest} from '@commonTests/BO/advancedParameters/newFeatures';
+
+// Import faker data
+import ProductFaker from '@data/faker/product';
+import files from '@utils/files';
+
+const baseContext = 'productV2_functional_CRUDStandardProduct';
```

Comment:
The filename is not correct : `CRUSStandardProduct.ts` => `CRUDStandardProduct.ts`

Reply:
;)